src/plugins/presentation_util/public/components/solution_toolbar/solution_toolbar.tsx

```diff
@@ -23,8 +22,9 @@ import './solution_toolbar.scss';
 interface NamedSlots {
   primaryActionButton: ReactElement<typeof PrimaryActionButton | typeof PrimaryActionPopover>;
   quickButtonGroup?: ReactElement<typeof QuickButtonGroup>;
-  addFromLibraryButton?: ReactElement<typeof AddFromLibraryButton>;
-  extraButtons?: Array<ReactElement<typeof SolutionToolbarButton | typeof SolutionToolbarPopover>>;
+  extraButtons?: Array<
+    ReactElement<typeof SolutionToolbarButton | typeof SolutionToolbarPopover> | undefined
+  >;
 }
```

Comment:
As per the discussion in https://github.com/elastic/kibana/pull/125728#discussion_r819061297, after getting approval from the @elastic/shared-ux team, the `Add to library` and `Controls` buttons were moved in to the more generic `extraButtons` array.